appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/callbacks/CallbackImpl.java

```diff
@@ -39,29 +39,52 @@
  */
 package fish.payara.microprofile.openapi.impl.model.callbacks;
 
+import fish.payara.microprofile.openapi.api.visitor.ApiContext;
+import fish.payara.microprofile.openapi.impl.model.ExtensibleTreeMap;
+import fish.payara.microprofile.openapi.impl.model.OperationImpl;
+import fish.payara.microprofile.openapi.impl.model.PathItemImpl;
 import static fish.payara.microprofile.openapi.impl.model.util.ModelUtils.applyReference;
 import static fish.payara.microprofile.openapi.impl.model.util.ModelUtils.getHttpMethod;
 import static fish.payara.microprofile.openapi.impl.model.util.ModelUtils.getOrCreateOperation;
-import static fish.payara.microprofile.openapi.impl.model.util.ModelUtils.isAnnotationNull;
-
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Map;
-
-import org.eclipse.microprofile.openapi.annotations.callbacks.CallbackOperation;
 import org.eclipse.microprofile.openapi.models.Operation;
 import org.eclipse.microprofile.openapi.models.PathItem;
 import org.eclipse.microprofile.openapi.models.PathItem.HttpMethod;
 import org.eclipse.microprofile.openapi.models.callbacks.Callback;
-import org.eclipse.microprofile.openapi.models.media.Schema;
-
-import fish.payara.microprofile.openapi.impl.model.ExtensibleTreeMap;
-import fish.payara.microprofile.openapi.impl.model.OperationImpl;
-import fish.payara.microprofile.openapi.impl.model.PathItemImpl;
+import org.glassfish.hk2.classmodel.reflect.AnnotationModel;
 
 public class CallbackImpl extends ExtensibleTreeMap<PathItem, Callback> implements Callback {
 
     private static final long serialVersionUID = 5549098533131353142L;
 
     private String ref;
+    
+    private String urlExpression;
+
+    private List<Operation> operations;
```

Comment:
I am surprised to see addition of these fields. Is this adding a feature that wasn't there before?

CC @MattGill98

Reply:
Few OpenAPI Annotations and Models class  (e.g Callback, Content, etc) differ in attributes but not the end results.
As earlier merge operation performed from Annotation (from) instance to Model (to), these attributes storage is not required. But with this PR HK2 types converted to Model (from) and merge operation is performed, hence locally these additional attributes are required.